tap_google_sheets/client.py

```diff
@@ -116,12 +116,12 @@ def raise_for_error(response):
                 # There is nothing we can do here since Google has neither sent
                 # us a 2xx response nor a response content.
                 return
+            status_code = response.status_code # Get status code of the response
             response = response.json()
             if ('error' in response) or ('errorCode' in response):
-                message = '%s: %s' % (response.get('error', str(error)),
-                                      response.get('message', 'Unknown Error'))
-                error_code = response.get('error', {}).get('code')
-                ex = get_exception_for_error_code(error_code)
+                message = 'HTTP-error-code: %s %s: %s' % (status_code, response.get('error', str(error)),
```

Comment:
In previous, if 'message' does not found in JSON response then it was showing directly Unknown Error in the error message. But, we observed that in many responses error_description comes as a detailed cause of the error. So, here we added an extra check for error_description to better readability. If error_description also does not found then it shows Unknown Error.